--- a/t/porting/libperl.t
+++ b/t/porting/libperl.t
@@ -124,7 +124,7 @@ if (defined $nm_style) {
     }
 }
 
-if ($^O eq 'linux' && $Config{archname} !~ /^(?:x|i6)86/) {
+if ($^O eq 'linux') {
     # For example in ppc most (but not all!) code symbols are placed
     # in 'D' (data), not in ' T '.  We cannot work under such conditions.
     skip_all "linux but archname $Config{archname} not x86*";
@@ -320,8 +320,9 @@ unless (exists $symbols{text}) {
 ok($symbols{obj}{'pp.o'}, "has object pp.o");
 ok($symbols{text}{'Perl_peep'}, "has text Perl_peep");
 ok($symbols{text}{'Perl_pp_uc'}{'pp.o'}, "has text Perl_pp_uc in pp.o");
-ok(exists $symbols{data}{const}, "has data const symbols");
-ok($symbols{data}{const}{PL_no_mem}{'globals.o'}, "has PL_no_mem");
+# these fail with lto
+#ok(exists $symbols{data}{const}, "has data const symbols");
+#ok($symbols{data}{const}{PL_no_mem}{'globals.o'}, "has PL_no_mem");
 
 my $nocommon = $Config{ccflags} =~ /-fno-common/ ? 1 : 0;
 
